### PR TITLE
[BPK-1826] Create per subspec `README.md`s

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: Backpack/Shadow/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Usage
+
+`Backpack/Shadow` contains the Backpack shadow definitions in the class `BPKShadow`.
+
+```objective-c
+#import <Backpack/Shadow.h>
+
+BPKShadow *shadowSm = [BPKShadow shadowSm];
+// BPKShadow *shadowLg = [BPKShadow shadowLg];
```
Comment (matthewdavidson) [inline Backpack/Shadow/README.md]: why are these commented out?

Comment (k0nserv) [reply Backpack/Shadow/README.md]: Just to illustrate that they are there, makes it easier to copy the code

File: Backpack/Gradient/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+pod 'Backpack/Gradient'
+```
+
+and then run `pod install`.
+
+## Usage
+
+`Backpack/Gradient` contains the Backpack gradient definitions in the class `BPKGradient`. `BPKGradient` has the two properties:
+
+* `colors` of type `NSArray<UIColor *> *`
```
Comment (tiagoengel) [inline Backpack/Gradient/README.md]: Can we move props to a different section like we do with react/react native components?

Comment (k0nserv) [reply Backpack/Gradient/README.md]: I think we should just remove these all together in favour of API docs in code

**Pull request merged**